### ISSUE TITLE
Feature: Autoload from Path

### ISSUE THREAD

**Issue by dreamsyntax** [aldelaro5/dolphin-memory-engine#121]
While the automatic saving/loading of the watchlist from a temp file in appdata is useful, I find myself losing work in a multi user dmw if I forget to save.

- allow auto load/save on close to use a last opened/saved file (only when a persistant setting is set)
- prompt on closing? If temp file is being used.

**Comment by cristian64** [comment]
As of #135, users will at least be warned, and will have a chance to save the file on exit if it has unsaved changes.

**Comment by dreamsyntax** [comment]
#135 resolves issues with multiple DME instances - now merged.

Attached in a below use case I want to address before we can close this issue.

1. shared git repo with DMWs in it
2. open the DMW directly with DME, make changes. Save and quit. Commit changes. A day goes by, other people are working on the same DMW and add updates.
3. pull from git repo as someone updated it.
4. Later open DME, and since it auto keeps the changes from last DMW load, I forget to re-load from the file in the git repo (thus losing the changes from the other user's commits)

Effectively, I'm looking for an auto read from specific file directory mode rather than saving to/from the dolphin-memory-engine.ini.

**Comment by cristian64** [comment]
In some offline discussion, these actions can be added to the **File** menu:
```
File > [x] Autoload Last File
     > [x] Autosave Last File
```
> C: Second option only enabled if first option is enabled, perhaps?
> D: I like it!
> C: If only first option is enabled, the prompt is shown.
> D: Yeah, that sounds ideal.

**Comment by dreamsyntax** [comment]
Finally getting around to implementing this as it's become a pain not having it in my workflow.

![Image](https://github.com/user-attachments/assets/7ba1166c-a881-451e-ab42-be30bb9c819a)

I'm not sure what it is, but the text looks off...
Open to opinions on `Autosave` vs `Auto-save` vs `Auto Save`

**Comment by dreamsyntax** [comment]
Nearing completion of Autoload.
After initial usage I'm thinking Autosave is a bit too risky/annoying without https://github.com/aldelaro5/dolphin-memory-engine/issues/182 implemented first.
Will release Autoload by itself, with closing prompting the existing unsaved changes, I think it may be overkill anyway.

**Comment by dreamsyntax** [comment]
I'm definitely thinking Auto-save is overkill and somewhat dangerous, so unless someone wants that in the future, I won't make a new issue for it.

**Comment by dreamsyntax** [comment]
Released in v1.2.5
